src/Xrefcheck/Config.hs

```diff
@@ -11,9 +11,11 @@ import Data.Aeson.Options (defaultOptions)
 import Data.Aeson.TH (deriveFromJSON)
 import Data.Yaml (FromJSON (..), decodeEither', prettyPrintParseException, withText)
 import Instances.TH.Lift ()
-import qualified Language.Haskell.TH.Syntax as TH
-import System.FilePath ((</>))
-import TH.RelativePaths (qReadFileBS)
+
+-- FIXME: Use </> from System.FilePath
```

Comment:
Ermm, looks like a big hack. How are we going to fix it? If we import from Posix, why not use `/` directly?

Reply:
We import from POSIX specifically so that when a fix comes, it's a small diff.

I think we have to write our own implementation of `</>` at some point that's `/` if build == POSIX and `\` if build == Windows. I don't know how to do that properly though.

Another solution would be to place the config at the root of the repository so that we don't need `</>` at all :P

Reply:
> I think we have to write our own implementation of `</>` at some point that's `/` if build == POSIX and `\` if build == Windows.

Shouldn't it be handled by `filepath`? Maybe it's worth creating an issue there?

Reply:
It probably should, but I believe that getting the build platform will be GHC-specific.